diff --git a/makefiles/common.mk b/makefiles/common.mk
index 35d1826..fd33155 100644
--- a/makefiles/common.mk
+++ b/makefiles/common.mk
@@ -21,7 +21,13 @@ CUDA_VERSION = $(strip $(shell which $(NVCC) >/dev/null && $(NVCC) --version | g
 #CUDA_VERSION ?= $(shell ls $(CUDA_LIB)/libcudart.so.* | head -1 | rev | cut -d "." -f -2 | rev)
 CUDA_MAJOR = $(shell echo $(CUDA_VERSION) | cut -d "." -f 1)
 CUDA_MINOR = $(shell echo $(CUDA_VERSION) | cut -d "." -f 2)
-#$(info CUDA_VERSION ${CUDA_MAJOR}.${CUDA_MINOR})
+NVCC_RELEASE = $(strip $(shell which $(NVCC) >/dev/null && $(NVCC) --version | grep release))
+$(info NVCC_RELEASE ${NVCC_RELEASE})
+$(info CUDA_VERSION ${CUDA_MAJOR}.${CUDA_MINOR})
+$(info CUDA_VERSION ${CUDA_VERSION} ${CUDA_MAJOR}.${CUDA_MINOR})
+
+# Avoid complete build for now
+exit 1
 
 # You should define NVCC_GENCODE in your environment to the minimal set
 # of archs to reduce compile time.
